### PR TITLE
Rewrite the 'scope of a track' document

### PR DESCRIPTION
**Pull request by kytrinyx**
@rpottsoh noted in https://github.com/exercism/discussions/issues/206 that this document was confusing.

Having reread it I wholeheartedly agree!

This takes a stab at rewriting it.

### CODE REVIEW

File: about/scope-of-a-track.md [hunk 1]
```diff
@@ -1,30 +1,62 @@
...
+* **fluency:** ease and familiarity, not having to think about syntax
+* **idioms:** using the conventions that are common for the language
 
-This sparked a very lively discussion with lots of excellent points about what the purpose of Exercism is, the value and danger of relying on 3rd party libraries, and whether or not the introduction of this type of Exercise would change the platform so much that it would be unrecognizable. It's a great discussion, and you should go read it.
+Familiarity with third party libraries is notably absent from this list.
+As is the ability to write scalable, production-worthy code and solve real-world problems.
 
-The question of what should be included or excluded on Exercism has come up before. Brian Hicks wondered whether or not we should have a track for JSON in [exercism/discussions#84](https://github.com/exercism/discussions/issues/84), after observing that people do some funny things with it. I've received email on a couple of occasions, asking whether or not we could add HTML and/or CSS tracks.
+There are plenty of resources available to do these things—but it's particularly challenging to make use of them without fluency in the basics.
```
Comment (rpottsoh) [inline about/scope-of-a-track.md]: What Things?  Is this statement speaking about familiarity with third party libraries? If so, this should probably not be a new paragraph and should be a part of the Familiarity statement above.

**Comment by rpottsoh** [comment]
@kytrinyx this is a lot better, thanks for making these changes.

File: about/scope-of-a-track.md (hunk 1 above)
Comment (kytrinyx) [reply about/scope-of-a-track.md]: third party libraries and real world problems

Comment (kytrinyx) [reply about/scope-of-a-track.md]: I'll take another pass at this part

**Pull request merged**